packages/core/src/middleware/koa-spa-proxy.ts

```diff
@@ -40,25 +47,21 @@ export default function koaSpaProxy<StateT, ContextT extends IRouterParamContext
 }: Properties): MiddlewareType<StateT, ContextT, ResponseBodyT> {
   type Middleware = MiddlewareType<StateT, ContextT, ResponseBodyT>;
 
-  const distributionPath = getDistributionPath(packagePath);
-
-  const spaProxy: Middleware = EnvSet.values.isProduction
-    ? serveStatic(distributionPath)
-    : proxy('*', {
```

Comment:
the ternary here avoids unnecessary code execution. the updated code will create a dev proxy anyway